### PR TITLE
Feat str replace

### PR DESCRIPTION
**Pull request by vektor8891**

### CODE REVIEW

**Comment by vektor8891** [comment]
@maxinteger I've created a new branch with the two requested functions. Can you pls take a look?

File: src/data/string/stringFn.ts [hunk 1]
```diff
@@ -7,3 +7,17 @@ export const plStringConstructor = (value: PLString): PLString => {
   typeCheck(PLString, value)
   return value
 }
+
+export const replace = (from: PLString, to: PLString, str: PLString): PLString => {
+  return new PLString(str.value.replace(from.value, to.value))
+}
+
+export const replaceRegexp = (regexp: PLString, flags: PLString, to: PLString, str: PLString): PLString => {
```
Comment (maxinteger) [inline src/data/string/stringFn.ts:15]: could you please change order of parameters to:
```suggestion
export const replaceRegexp = (flags: PLString, regexp: PLString, to: PLString, str: PLString): PLString => {
```
this way you can define replace function with default flags:
```ts
(let replace-global (replace-regexp "g"))
```

**Review by maxinteger — CHANGES_REQUESTED** [review]
Looks very good. I just need one tiny change.

**Comment by vektor8891** [comment]
@maxinteger Done. Can you pls take a look?

**Comment by vektor8891** [comment]
@maxinteger Can you pls deploy the newest version? I still cannot use this function on the website.

**Pull request merged**